src/main/java/seedu/address/model/person/Name.java

```diff
@@ -13,7 +13,7 @@ public class Name {
             "Names should only contain alphanumeric characters and spaces, and it should not be blank";
 
     /*
-     * The first character of the address must not be a whitespace,
+     * The first character of the title must not be a whitespace,
```

Comment:
this should be name right?

Reply:
Yes, it just has to be a string.

Reply:
as in, the comment should say 'name' instead of 'title' right?

Reply:
Ah crap.